app/client/src/pages/_document.tsx

```diff
@@ -8,9 +8,24 @@ export default class MyDocument extends Document {
         return (
             <Html lang='en'>
                 <Head>
+                    {/* <!-- Global site tag (gtag.js) - Google Analytics --> */}
+                    <script
+                        async
+                        src={`https://www.googletagmanager.com/gtag/js?id=${process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS}`}
+                    />
+                    <script
+                        dangerouslySetInnerHTML={{
+                            __html: `
+                                window.dataLayer = window.dataLayer || [];
+                                function gtag(){dataLayer.push(arguments);}
+                                gtag('js', new Date());
+                                gtag('config', '${process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS}', { page_path: window.location.pathname });
+                                `,
+                        }}
```

Comment:
Hmm, why are we dangerously setting inner html.  Should almost never be necessary.